spec/support/pages/page.rb

```diff
@@ -75,7 +75,6 @@ def view_card(card_name, overlay_class=nil, &block)
     retry_stale_element do
       find('.card-content', text: card_name).click
     end
-    synchronize_content! 'CLOSE'
```

Comment:
@mikem Line 84's `find()` also waits for the card to be rendered before continuing. In practice finding by css can also be quite a bit quicker than using `synchronise_content` on a big page.  All of our overlays have both 'close' button and the '.overlay' class, so checking for both felt redundant.

84: `overlay = overlay_class.new session.find(".overlay")`